services/web/server/src/simcore_service_webserver/projects/projects_models.py

```diff
@@ -77,44 +77,64 @@ def _convert_to_schema_names(project_db_data) -> Dict:
     return converted_args
 
 class ProjectDB:
+    # TODO: should implement similar model as services/web/server/src/simcore_service_webserver/login/storage.py

```

Comment:
which models? you mean db_models.py? more information there please.

Reply:
i mean that this should look similar to the design in the login. The class with high-level calls to the db is called storage. It would be good to have the same naming and a consistent functionality ... otherwise it is going to be very difficult to deal with the webserver

Again, something to cleanup